From: Bruce Rogers <brogers@suse.com>
Date: Wed, 2 Oct 2019 07:28:04 -0600
Subject: tests: Disable some block tests for now

Most tests previously disabled for qemu-testsuite to be able to complete
successfully are no longer (as of v4.1) listed as auto, and therefore
do not get run anymore.

27NOV2019 - added 161 since it is failing on s390x and ppc consistently

Signed-off-by: Bruce Rogers <brogers@suse.com>
---
 tests/qemu-iotests/group | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/tests/qemu-iotests/group b/tests/qemu-iotests/group
index 435dccd5af90754a3f80f4455bdb..a6c8ed8a5cdef50383594305c0c1 100644
--- a/tests/qemu-iotests/group
+++ b/tests/qemu-iotests/group
@@ -182,7 +182,7 @@
 158 rw auto quick
 159 rw auto quick
 160 rw quick
-161 rw auto quick
+#DISABLE FOR NOW 161 rw auto quick
 162 quick
 163 rw
 165 rw quick
